examples/example_internal_limits.html

```diff
@@ -0,0 +1,168 @@
+        require_consent: true, //this true means consent is required
+      });
+
+      //(optionally) provide custom feature tree if needed
+      Countly.group_features({
+        activity: ["sessions", "events", "views"],
+      });
+
+      //we can call all the helper methods we want, they won't record until consent is provided for specific features
+      Countly.track_sessions();
+      //we can check event key, value lengths and segmentation numbers with view event logs here too
+      Countly.track_pageview();
+
+      //Consent Management logic should be implemented and controlled by developer
+      //this is just a simply example of what logic it could have
+      if (typeof localStorage !== "undefined") {
+        var consents = localStorage.getItem("consents");
+        //checking if user already provided consent
+        if (consents) {
+          //we already have array with consents from previous visit, let's just pass them to Countly
+          Countly.add_consent(JSON.parse(consents));
+        } else {
+          //user have not yet provided us a consent
+          //we need to display popup and ask user to give consent for specific features we want to track
+          //for example purposes, we will wait till user clicks "Give Consent" button
+          //to allow consent for "activity", "crashes"
+        }
+      } else {
+        // Sorry! No Web Storage support..
+        // we can fallback to cookie
+      }
+
+      //first give consent for features to work
+      function giveConsent() {
+        //give consent to necessary features
+        var response = ["activity", "crashes", "users"];
+        Countly.add_consent(response);
+        localStorage.setItem("consents", JSON.stringify(response));
+      }
+      //then you can test stack trace related limits here (line length and line per line)
+      function recordError() {
+        //create new error object with stack key
+        var error = {
+          stack: "Enter your\n error here\n separated\n with newlines\n",
+        };
+        Countly.recordError(error, true);
+      }
+      //you can add logs and check breadcrumb limits here (present crashLogs limit at a time)
+      function addLog() {
+        //add new log into crashLogs
+        var log = "Enter your log here";
+        Countly.add_log(log);
+      }
+
+      //add user details
+      function addUserDetails() {
+        Countly.user_details({
+          name: "Arturs Sosins",
+          username: "ar2rsawseen",
+          email: "test@test.com",
+          organization: "Countly",
+          phone: "+37112345678",
+          //Web URL pointing to user picture
+          picture:
+            "https://pbs.twimg.com/profile_images/1442562237/012_n_400x400.jpg",
+          gender: "M",
+          byear: 1987, //birth year
+          custom: {
+            key1: "value1",
+            key2: "value2",
+          },
+        });
+      }
+      //modify user data
+      function set() {
```

Comment:
these modification calls should be reworked so that each of them breaks the set limits